extensions/opentelemetry/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/TracerUtilTest.java

```diff
@@ -0,0 +1,63 @@
+package io.quarkus.opentelemetry.deployment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.trace.data.LinkData;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import io.opentelemetry.sdk.trace.samplers.SamplingResult;
+import io.quarkus.opentelemetry.runtime.tracing.DropNamesSampler;
+import io.quarkus.opentelemetry.runtime.tracing.TracerRuntimeConfig;
+import io.quarkus.opentelemetry.runtime.tracing.TracerUtil;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class TracerUtilTest {
```

Comment:
Please look into the `NonAppEndpoints*Test's` and implement a similar test for a custom sampler in `OpenTelemetrySamplerBeanTest`.

Reply:
updated OpenTelemetrySamplerBeanTest to use custom sampler

Reply:
The idea was to implement a new test, not change the current one. With this approach, we are not testing the previous scenario when there is no custom CDI Sampler.

Can you please keep the current test and make a new one on the side with the custom CDI Sampler? I'm sorry if I was not clear. Thanks!

Reply:
sorry, I had the impression that the initial test was testing that a sampler bean is accesible via openTelemetry when defined in the Application context, thus could be replaced with a custom one 
Reverted these changes and added as a separate test